sklearn/metrics/cluster/_supervised.py

```diff
@@ -828,10 +829,10 @@ def adjusted_mutual_info_score(labels_true, labels_pred, *,
     Parameters
     ----------
     labels_true : int array, shape = [n_samples]
-        A clustering of the data into disjoint subsets.
+        A clustering of the data into disjoint subsets (U).
```

Comment:
refer to the formula

Reply:
All of these make sense to me. I just pushed an update that should incorporate all of these. Thanks a lot!